tests/unit/Codeception/Module/AssertsTest.php

```diff
@@ -66,8 +71,60 @@ public function testExceptionFails()
      */
     public function testOutputExceptionTimeWhenNothingCaught()
     {
-        $module = new \Codeception\Module\Asserts(make_container());
-        $module->expectException(RuntimeException::class, function () {
+        $this->module->expectException(RuntimeException::class, function () {
+        });
+    }
+
+    public function testExpectThrowable()
+    {
+        $this->module->expectThrowable('Exception', function () {
+            throw new Exception();
+        });
+        $this->module->expectThrowable(new Exception('here'), function () {
+            throw new Exception('here');
+        });
+        $this->module->expectThrowable(new Exception('here', 200), function () {
+            throw new Exception('here', 200);
+        });
+    }
+
+    /**
+     * @expectedException PHPUnit\Framework\AssertionFailedError
+     */
+    public function testExpectThrowableFailOnDifferentClass()
+    {
+        $this->module->expectThrowable(new RuntimeException(), function () {
+            throw new Exception();
+        });
+    }
+
+    /**
+     * @expectedException PHPUnit\Framework\AssertionFailedError
+     */
+    public function testExpectThrowableFailOnDifferentMessage()
+    {
+        $this->module->expectThrowable(new Exception('foo', 200), function () {
+            throw new Exception('bar', 200);
+        });
+    }
+
+    /**
+     * @expectedException PHPUnit\Framework\AssertionFailedError
```

Comment:
Please don't use annotations https://github.com/sebastianbergmann/phpunit/issues/3332

Reply:
Fixed in f8a2b5289cc907bb095ee0be889d1fa86ef086af